packages/spotlight/src/spotlight.js

```diff
@@ -249,17 +250,20 @@ const Spotlight = (function () {
 
 	function restoreFocus () {
 		const lastContainerId = getLastContainer();
-		const next = [rootContainerId];
+		let next;
 
 		if (lastContainerId) {
-			next.unshift(lastContainerId);
+			// walk up the chain of containers from the last to attempt to find a target
+			next = getContainersForNode(getContainerNode(lastContainerId)).reverse();
```

Comment:
Should we test for an empty array here (or at the end of this) to set `next` to `[rootContainerId]`?

Reply:
getContainersForNode will always contain the root container id.

Reply:
Argh, misread the source.  :nothingtodohere: